src/snapshot.zig

```diff
@@ -346,16 +346,7 @@ fn processSnapshotFile(gpa: Allocator, snapshot_path: []const u8, maybe_fuzz_cor
 
         // create a pseudo-random name for our file
         const rand_file_name = [_][]const u8{
-            maybe_fuzz_corpus_path.?, &[_]u8{
-                rand.intRangeAtMost(u8, 'a', 'z'),
-                rand.intRangeAtMost(u8, 'a', 'z'),
-                rand.intRangeAtMost(u8, 'a', 'z'),
-                rand.intRangeAtMost(u8, 'a', 'z'),
-                rand.intRangeAtMost(u8, 'a', 'z'),
-                rand.intRangeAtMost(u8, 'a', 'z'),
-                rand.intRangeAtMost(u8, 'a', 'z'),
-                rand.intRangeAtMost(u8, 'a', 'z'),
-            },
+            maybe_fuzz_corpus_path.?, &[_]u8{ rand.intRangeAtMost(u8, 'a', 'z'), rand.intRangeAtMost(u8, 'a', 'z'), rand.intRangeAtMost(u8, 'a', 'z'), rand.intRangeAtMost(u8, 'a', 'z'), rand.intRangeAtMost(u8, 'a', 'z'), rand.intRangeAtMost(u8, 'a', 'z'), rand.intRangeAtMost(u8, 'a', 'z'), rand.intRangeAtMost(u8, 'a', 'z'), '.', 'r', 'o', 'c' },
```

Comment:
nit: re-add trailing comma to make this pretty again

Reply:
done in 300267cad8